nexusutils/dataconverter/readers/json_map/reader.py

```diff
@@ -32,12 +33,7 @@ def parse_slice(slice_string):
     slices = slice_string.split(",")
     for index, item in enumerate(slices):
         values = item.split(":")
-        if len(values) == 1:
-            slices[index] = int(values[0])
-        else:
-            if len(values) < 3:
-                values.append("")
-            slices[index] = slice(map(int, filter(lambda x: not x, values)))
+        slices[index] = slice(*[None if x == '' else int(x) for x in values])
```

Comment:
@domna This works for now. I tried "::x", "x:", and ":x:". They all seem to work.